packages/consent-manager/components/dialog.module.css

```diff
@@ -13,6 +13,7 @@
   justify-content: center;
   align-items: center;
   z-index: 10;
+  background-color: transparent;
```

Comment:
Is this doing something accessibility-wise?

Reply:
No, but with switching the container to a <dialog> our normalize.css file has it's background color as white. This overrides that.

Reply:
Makes sense 👍